src/ekomark/benchmark/external/LHA_utils.py

```diff
@@ -129,16 +129,22 @@ def compute_LHA_data(
         ref : dict
             output containing: target_xgrid, values
     """
-
+    Polarized= operators["polarized"]
     Q2grid = operators["Q2grid"]
     if not np.allclose(Q2grid, [1e4]):
         raise ValueError("Q2grid has to be [1e4]")
+    order = theory["PTO"]
+    # select which data
+    if Polarized == True and order <= 1:
```

Comment:
```suggestion
    if Polarized and order <= 1:
```
no need to check a boolean again :upside_down_face: